Tests/WorkspaceTests/WorkspaceTests.swift

```diff
@@ -12355,6 +12355,25 @@ final class WorkspaceTests: XCTestCase {
     }
 }
 
+final class WorkspacePerformanceTests: XCTestCasePerf {
```

Comment:
perhaps we should move this to `PackageGraphPerfTests`? @neonichu opinions?

Reply:
Yah, I think testing that at the package graph rather than the workspace level sounds right to me, but I am not sure we have infra there to load from fixtures.

Reply:
`fixture` is in SPMTestSupport so should be fine? if not, we can leave it here

Reply:
Once I looked at testing the package graph more directly instead of via the workspace, doing it with generated Manifests rather than fixtures seemed viable, and more in keeping with the other Package tests.

Updated.